<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConditionBox</class>
 <widget class="QWidget" name="ConditionBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="whenComboBox">
     <item>
      <property name="text">
       <string>WHEN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WHEN NOT</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>AND</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XOR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AND NOT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OR NOT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XOR NOT</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="ExpressionLineEdit" name="expressionBox">
     <property name="placeholderText">
      <string>Expression to check</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="list-remove"/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpressionLineEdit</class>
   <extends>QLineEdit</extends>
   <header>expressionlineedit.h</header>
   <slots>
    <signal>expressionUpdated(QString)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
